components/Challenge/ProChallengeContent.tsx

```diff
@@ -41,144 +48,198 @@ const ProChallengeContent: React.FC<ProChallengeContentProps> = ({
+                    <p>Given this scenario, we have two pantheons:</p>
+                    <ul>
+                        <li>
+                            Pantheon 1: Zeus, Apollo, Athena, with a total
+                            divine power of 102.
+                        </li>
+                        <li>
+                            Pantheon 2: Hades, Hermes, and Artemis, with a total
+                            divine power of 87.
+                        </li>
+                    </ul>
+                    <p>Therefore, the max_divine_power is 102.</p>
+                </div>
+                <div className={styles.block}>
+                    <h3>Constraints</h3>
+                    <ul>
+                        <li>
+                            The input edges (alliances) and nodes (gods/heroes)
+                            will be valid.
+                        </li>
+                        <li>
+                            Divine power values can be positive or negative
+                            integers, as gods and heroes can be either
+                            benevolent or malevolent.
+                        </li>
+                        <li>The graph will not contain any cycles.</li>
+                        <li>
+                            Your celestial JWT token will never expire, so you
+                            can take as long as needed to complete the
+                            challenge.
+                        </li>
+                    </ul>
+                </div>
+                <div className={styles.block}>
+                    <h3>Notes</h3>
+                    <ul>
+                        <li>
+                            We will monitor the number of submissions to
+                            discourage brute force methods. You have unlimited
+                            attempts, but be strategic in your submissions.
+                        </li>
+                        <li>
+                            Use any resources you need, but your work must be
+                            your own. Collaboration is not allowed.
+                        </li>
+                        <li>
+                            Completing the challenge qualifies you to apply as
+                            an Olympian, but it does not guarantee admission.
+                        </li>
+                    </ul>
+                </div>
 
-            <div className={styles.finishedChallenge}>
-                <ShineButton
-                    text="Finished Challenge"
-                    onClick={handleUserFinishedChallenge}
+                <div className={styles.links}>
+                    <p onClick={handleSuccess} className={styles.link}>
+                        <a>Test success case</a>
+                    </p>
+                    <p onClick={handleFailure} className={styles.link}>
+                        <a>Test failure case</a>
+                    </p>
+                </div>
```

Comment:
This probably shouldn't go into production

Reply:
I left this for testing purposes but feel free to remove it / let me know to remove it when ready